Resources/Maps/_NF/Shuttles/akupara.yml

```
@@ -308,65 +342,60 @@ entities:
             color: '#CC5100FF'
             id: MiniTileWhiteCornerSe
           decals:
-            20: 5,5
-            30: 7,7
+            87: 8,7
         - node:
             color: '#CC5100FF'
             id: MiniTileWhiteCornerSw
           decals:
             19: 3,7
-            21: 4,5
-        - node:
-            color: '#CC5100FF'
-            id: MiniTileWhiteInnerSe
-          decals:
-            28: 5,7
-        - node:
-            color: '#CC5100FF'
-            id: MiniTileWhiteInnerSw
-          decals:
-            29: 4,7
         - node:
             color: '#CC5100FF'
             id: MiniTileWhiteLineE
           decals:
-            24: 5,6
-            31: 7,8
+            88: 8,8
         - node:
             color: '#CC5100FF'
             id: MiniTileWhiteLineN
           decals:
-            25: 4,9
             26: 5,9
             27: 6,9
+            91: 7,9
+            160: 4,9
         - node:
             color: '#CC5100FF'
             id: MiniTileWhiteLineS
           decals:
-            69: 6,7
+            90: 7,7
+            95: 5,7
+            105: 6,7
         - node:
             color: '#CC5100FF'
             id: MiniTileWhiteLineW
           decals:
             22: 3,8
-            23: 4,6
         - node:
             cleanable: True
             color: '#FFFFFFFF'
             id: Rock04
           decals:
             8: -0.98000634,-5.253206
         - node:
-            angle: 4.71238898038469 rad
             color: '#FFFFFFFF'
             id: StandClear
           decals:
-            82: 6,-10
+            111: 6,4
+        - node:
+            angle: 1.5707963267948966 rad
+            color: '#FFFFFFFF'
+            id: StandClear
+          decals:
+            116: 7,5
         - node:
+            angle: 4.71238898038469 rad
```

```suggestion
            angle: -1.5707963267948966 rad
```
Not sure how you managed that.